public/js/io/model.js

```diff
@@ -140,13 +140,23 @@ angular.module('app').service('ioModel', function (connection, $q) {
     this.importBundle = function (bundle, datasourceRef) {
         const additions = [];
         const messages = [];
+        const replace = [];
 
         const layerPromises = [];
         for (const layer of bundle.layers) {
             const p = getLayer(layer._id).then(l => {
                 if (l) {
-                    messages.push('Layer was not imported because it already exists in database: ' + l.name);
-                    return;
+                    if (layer.replace === true) {
+                        var msg = gettextCatalog.getString('This layer was replaced: ');
```

Comment:
Strings passed to `gettextCatalog.getString` are trimmed when extracted to the .pot file. They should contain no leading or trailing whitespaces.